scripts/splitSources.py

```diff
@@ -40,21 +40,21 @@ def writeSourceToFile(lines):
     filePath, srcName = extractSourceName(lines[0])
     # print("sourceName is ", srcName)
     # print("filePath is", filePath)
-    if filePath != False:
+    if filePath != False and os.path.isfile(srcName):
```

Comment:
This is required because the test case added by this PR leads to an error here (`srcName` is actually a directory in python that looks like `///`)

Reply:
Looks like this change has been removed.